src/celestia/hud.cpp

```diff
@@ -22,6 +22,18 @@
 
 #include <fmt/format.h>
 
+#ifdef USE_ICU
+#ifdef HAVE_WIN_ICU_COMBINED_HEADER
+#include <icu.h>
+#elif defined(HAVE_WIN_ICU_SEPARATE_HEADERS)
+#include <icucommon.h>
+#include <icui18n.h>
+#else
+#include <unicode/uloc.h>
+#include <unicode/ulocdata.h>
+#endif
+#endif
+
```

Comment:
can we have celutil/icu.h file instead of duplicating this block in multiple files?